src/gc/impl/conservative/gc.d

```diff
@@ -2883,8 +2921,6 @@ struct Gcx
         cstdlib.free(scanThreadData);
         // scanThreadData = null; // keep non-null to not start again after shutdown
         numScanThreads = 0;
-        version (Posix)
-            forkedGcx = null;
```

Comment:
I think this cleanup and the check in the callback should be kept in case the runtime is started and stopped manually and a fork call happens later.

Reply:
You're right, forgot about that. Fixed.